OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -84,6 +62,7 @@ public class UpdatePath
 				new MoveClassicFacingFudge(),
 				new RenameWithNukeLaunch(),
 				new SpawnActorPowerDefaultEffect(),
+				new CopyIsometricSelectableHeight(),
```

Comment:
I intentionally left this out because it relies on the user having first extracted the original art.ini to the right location. My plan was that this would be run manually like the old health multiplication rules.

Reply:
Ok. Will remove it.

Reply:
But should it still remain without a versioned folder?

Reply:
I wasn't sure where to put it. Since it isn't in a path and we had the health multiplication rules outside as well I put it in the general folder.

Reply:
I was tossing up having this as an update rule at all, one could make the argument that it would be better as a utility command instead (except that this would require a bunch of duplication).

Having it unversioned seems like the most reasonable compromise for now, and we can always move it later if we come up with a better idea.